libs/sdk/src/lib/helpers/validate-kinetic-sdk-config.ts

```diff
@@ -10,21 +10,9 @@ function removeTrailingSlash(str: string) {
  * @returns {KineticSdkConfig}
  */
 export function validateKineticSdkConfig(config: KineticSdkConfig): KineticSdkConfig {
-  if (!config.endpoint) {
-    throw new Error(`validateKineticSdkConfig: no endpoint configured.`)
-  }
   if (!config.endpoint.startsWith('http')) {
     throw new Error(`validateKineticSdkConfig: the endpoint should start with http or https.`)
   }
-  if (!config.environment) {
-    throw new Error(`validateKineticSdkConfig: no environment configured.`)
-  }
-  if (!config.index) {
-    throw new Error(`validateKineticSdkConfig: no index configured.`)
-  }
-  if (!Number.isInteger(config.index)) {
-    throw new Error(`validateKineticSdkConfig: index should be an integer.`)
-  }
   return {
```

Comment:
@beeman I have removed these validations that you give me in the instructions, because TypeScript already handles them.

Reply:
The reason we want this is that TypeScript is not always used. Think for instance about dynamic configurstion of the SDK like in our demo app.

Reply:
@beeman thanks, makes sense. I added it back on with its tests and it is ready to review. :)